packages/paste-website/src/components/tokens-list/sectionIntros.tsx

```diff
@@ -0,0 +1,120 @@
+import {Callout, CalloutTitle, CalloutText} from '../callout';
+import {SiteLink} from '../SiteLink';
+import type {Token} from './types';
+
+const ColorGuidelinesLink: React.FC = () => <SiteLink to="/foundations/colors">Read the color guidelines.</SiteLink>;
+
+const BackgroundColorsIntro: React.FC = () => (
+  <Paragraph>
+    Background color tokens define the background color of a container. <ColorGuidelinesLink />
+  </Paragraph>
+);
+
+const BorderColorsIntro: React.FC = () => (
+  <Paragraph>
+    Border color tokens define the border color of a container. <ColorGuidelinesLink />
+  </Paragraph>
+);
+
+const BorderWidthsIntro: React.FC = () => (
+  <Paragraph>Border widths define the thickness of a border around a container.</Paragraph>
+);
+
+const BoxShadowsIntro: React.FC = () => (
+  <Box marginBottom="space80">
+    <Paragraph>
+      Box shadows are used in a number of ways: some are true shadows that are used to create depth and demonstrate the
+      hierarchy of layered elements; other box shadow tokens are duplicates of border tokens and are used to visually
+      add a border without affecting the height of the component; box shadow tokens are also used for focus states on
+      components.
+    </Paragraph>
+    <Callout variant="info">
+      <CalloutTitle as="h3">
+        <strong>Did you know?</strong>
+      </CalloutTitle>
+      <CalloutText>
+        Borders add height to an element, but box shadows do not. It&apos;s why we need to use box shadows on some
+        components and not others.
+      </CalloutText>
+    </Callout>
+  </Box>
+);
+
+const ColorsIntro: React.FC = () => (
+  <Paragraph>
+    Colors are tokens that do not have a defined usage in the system and can be used in custom compositions.{' '}
+    <ColorGuidelinesLink />
+  </Paragraph>
+);
+
+const FontsIntro: React.FC = () => (
+  <Paragraph>
+    Fonts define the different font families.
+    {/* Read the typography guidelines (TBA) */}
+  </Paragraph>
+);
+
+const FontSizesIntro: React.FC = () => (
+  <Paragraph>
+    Font sizes define the size of text. 1 rem unit = 16 pixels in Paste.
+    {/* Read the typography guidelines (TBA) */}
+  </Paragraph>
+);
+
+const FontWeightsIntro: React.FC = () => (
+  <Paragraph>Font sizes define the weight of text. Read the typography guidelines (TBA)</Paragraph>
+);
+
+const LineHeightsIntro: React.FC = () => (
+  <Paragraph>
+    Line heights define the height of a line of text. 1 rem unit = 16 pixels in Paste.
+    {/* Read the typography guidelines (TBA) */}
+  </Paragraph>
+);
+
+const RadiiIntro: React.FC = () => <Paragraph>Radii define the border radius of a container.</Paragraph>;
+
+const SizingsIntro: React.FC = () => (
+  <Paragraph>
+    Sizings define the size of UI elements.
+    {/* Read the spacing and layout guidelines (TBA)*/}
+  </Paragraph>
+);
+
+const SpacingsIntro: React.FC = () => <Paragraph>Spacings define the spacing between UI elements.</Paragraph>;
```

Comment:
I know these descriptions came from the designs, but.... it feels silly to say that sizings define sizes and spacings define spaces just for the sake of having category descriptions